go/worker/keymanager/worker.go

```diff
@@ -221,33 +220,26 @@ func (w *Worker) updateStatus(status *api.Status, runtimeStatus *runtimeStatus)
 		}
 	}()
 
-	// Initialize the key manager.
-	type InitRequest struct {
-		Checksum    []byte `json:"checksum"`
-		Policy      []byte `json:"policy"`
-		MayGenerate bool   `json:"may_generate"`
-	}
-	type InitCall struct { // nolint: maligned
-		Method string      `json:"method"`
-		Args   InitRequest `json:"args"`
-	}
-
 	var policy []byte
 	if status.Policy != nil {
 		policy = cbor.Marshal(status.Policy)
 	}
 
-	call := InitCall{
+	request := enclaverpc.Request{
```

Comment:
Hm, so this was a local call before as one must ensure that this method can only be called locally.